src/Lorentz/Contracts/BaseDAO.hs

```diff
@@ -53,6 +54,9 @@ baseDaoContract config@Config{..} = defaultContract $ contractName cDaoName $ do
       , #cToken_address /-> tokenAddress
       )
 
+simplestDaoContract :: Config () () -> Contract (BaseDAO.Parameter ()) (BaseDAO.Storage () ())
+simplestDaoContract = baseDaoContract
```

Comment:
Hm, I imagined that `defaultConfig` will be bound here, not only type parameters. This `defaultConfig` is what makes this DAO actually simplest. In another config someone may provide a more complicated proposal check or decision lambda thus making DAO not that simple.

Reply:
Ah okay, it makes sense to me, I thought you intentionally compared `trivialDAO` with  `baseDaoContract` because you wanted me to specialize the contract.

Reply:
I removed the commit with the previous attempt and pushed another one on top.

Reply:
We have `ciContract = GameDAO.gameDaoContract`, I imagined we would do the same for the trivial DAO, i. e. export a ready-to-use contract that doesn't need any function applications or type applications. We can even create `Lorentz.Contracts.TrivialDAO` and put this `trivialConfig` there like we do for `GameDAO`.